oauth2_http/java/com/google/auth/oauth2/ServiceAccountJwtAccessCredentials.java

```diff
@@ -300,23 +325,15 @@ public void getRequestMetadata(final URI uri, Executor executor,
           + "defaultAudience to be specified");
       }
     }
-    String assertion = getJwtAccess(uri);
-    String authorizationHeader = JWT_ACCESS_PREFIX + assertion;
-    List<String> newAuthorizationHeaders = Collections.singletonList(authorizationHeader);
-    return Collections.singletonMap(AuthHttpConstants.AUTHORIZATION, newAuthorizationHeaders);
-  }
-
-  /**
-   * Discard any cached data
-   */
-  @Override
-  public void refresh() {
```

Comment:
Assuming this is ok because google-auth-library-java version is < 1.x.x, but wanted to highlight just in case :)

Reply:
Nevermind. I see it just got moved.